float64[] command
string[] name
